packages/ember-htmlbars/lib/main.js

```diff
@@ -79,7 +67,6 @@ registerHelper('textarea', textareaHelper);
 registerHelper('collection', collectionHelper);
 registerHelper('each', eachHelper);
 registerHelper('unbound', unboundHelper);
-registerHelper('concat', concat);
```

Comment:
`concat` was always intended to be an internal interpolation hook- the version we use for the htmlbars hook should not be coupled to the implementation we use for a possible new helper.